uint8 start
uint8 objNum
